gitpage/app/models/gitpagedisplay.rb

```diff
@@ -0,0 +1,5 @@
+class Gitpagedisplay < ApplicationRecord
+    response = HTTParty.get('https://api.github.com/search/users?q=cdmackeyfree')
+    
```

Comment:
I see why you put this here, and there is definitely a way you could implement this using a model to handle the data. But since there's no database or logic, it's straightforward enough just to throw this one line into a controller action and render a single view.